src/rustdoc_cmd.rs

```diff
@@ -83,6 +83,9 @@ impl RustdocCommand {
         if all_features {
             cmd.arg("--all-features");
         }
+        if atty::is(atty::Stream::Stderr) {
+            cmd.arg("--color=always");
+        }
```

Comment:
Should this reference the `GlobalConfig` struct's `stderr` value instead of the global stderr, just in case they are different?

This might need a bit more plumbing.

Reply:
But isn't `anyhow::bail!` using the normal stderr, and that's where this command's output will go?

Reply:
https://github.com/obi1kenobi/cargo-semver-checks/blob/4de32032f5eeb73e9c7bc95dcc8b4dedc207e43a/src/rustdoc_cmd.rs#L92

Reply:
That is an excellent point!

We should change that too, before a "cargo-semver-checks-as-a-library" user opens an issue about their library polluting their stderr even though they configured its stderr behavior to be something else.

Reply:
I see. But nevertheless it should use the `GlobalConfig`, right?

Reply:
Done in 8f7d3870d6212ee2181dbb8c71f0bce004fe304f.